src/p4est_io.c

```diff
@@ -469,22 +469,22 @@ check_file_metadata (sc_MPI_Comm mpicomm, const char *filename,
   }
 
   /* check the format of the user string */
-  if (metadata[47] != '\n') {
+  if (metadata[79] != '\n') {
```

Comment:
Can we use the `#define`s here? Otherwise the code will break when we change it.

Reply:
Yes, if we introduce additional macros for the entries of the file header. I did this to make `check_file_metadata` independent of the size of the entries of the user header.

Reply:
Thanks, sounds good.